### DISCUSSION TITLE
Multiple json files

### DISCUSSION THREAD

**Discussion by cajohntd** [cmu-sei/GHOSTS#69] [General]
I would like to have multiple timeline files so I can then execute at different times.  Is there a way to make happen?  When I try to rename one to timeline2.json it does not read it.  

Also, having trouble getting the timeline to execute at a certain time.  I set the "UtcTimeOn" to the time I want but it execute as soon as I start ghosts.

**Comment by sei-dupdyke** [comment]
Can you post up your timeline that is not executing to your `UtcTimeOn` setting, please? With that, I can do some debugging to figure out the issue with the start time.

The client does not currently support multiple timelines in that way, although that is something that could be added in the future since every handler within a timeline gets its own thread anyway. As it is now, the client looks only for `timeline.json` as its default set of activities to perform. 

There are a couple of ways to send it "do this now" types of activities on top of the default `timeline.json`, but I'm not sure that helps you accomplish the same thing you are trying with multiple timelines.

**Comment by cajohntd** [comment]
{
    "Status": "Run",
    "TimeLineHandlers": [
        {
            "HandlerType": "Command",
            "Initial": "",
            "UtcTimeOn": "00:00:00",
            "UtcTimeOff": "24:00:00",
            "Loop": "False",
            "TimeLineEvents": [
                {
                    "Command": "net use x: \\\\server\\c$ /user:user password",
                    "CommandArgs": [
                        "del c:\\exercise\\ghosts\\chromedriver.exe",
						"copy x:\\workspace\\chromedriver.exe c:\\exercise\\ghosts\\chromedriver.exe",
						"exit"
                    ],
                    "DelayAfter": 50000,
                    "DelayBefore": 0
                }
            ]
        },
        {
            "HandlerType": "BrowserChrome",
			"Initial": "about:blank",
            "UtcTimeOn": "12:25:00",
            "UtcTimeOff": "20:00:00",
            "Loop": "True",
           "TimeLineEvents": [

                {
                    "Command": "random",
                    "CommandArgs": [
                        "http://www.ceoexpress.com",
                        "http://wikipedia.org",
                        "http://ebay.com",
                        "http://craigslist.org",
                        "http://reddit.com",
                        "http://instagram.com",
						],
                    "DelayAfter": 150000,
                    "DelayBefore": 900000
                }
			]
        }
    ]
}

-- I want this to copy the chromedriver file before it starts chrome, but chrome starts before the driver is copied.

**Comment by sei-dupdyke** [comment]
Aha, I see what you mean - the problem is that the handler starts up the application and then does nothing until the correct time period, which conflicts with your copying the executable.

For right now, the only thing I can think of is to separate the jobs into two separate timelines, and have the first one execute, and then copy in the second via a scheduled task or something once the first is complete.
